lstchain/image/muon/muon_analysis.py

```diff
@@ -274,8 +271,8 @@ def analyze_muon_event(subarray, tel_id, event_id, image, good_ring_config, plot
             x[dist_ringwidth_mask], y[dist_ringwidth_mask],
             image[dist_ringwidth_mask],
             muonringparam.radius,
-            muonringparam.center_x,
-            muonringparam.center_y,
+            muonringparam.center_fov_lon,
```

Comment:
aren't the units of these coordinates changing? I do not exactly recall, but I think that `muonringparam.center_x` was in `m` and  `muonringparam.center_fov_lon` looks like to be in degrees. The second one also sounds like sky coordinates instead of camera ones, but it may not be the case

Reply:
No, this was inconsistent before in ctapipe. it was already in deg before, but named x and y.